src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -111,7 +112,7 @@ public static void fillWorkspaceWithTestData(Workspace workspace) {
 		VariableElement sampleVariable1 = new VariableElement();
 		sampleVariable1.setName("test");
 		sampleVariable1.setValue("true");
-		sampleVariable1.setType(VariableElementType.LOGIC);
+		sampleVariable1.setType(VariableElementTypeLoader.LOGIC);

```

Comment:
Maybe these should be moved to subclass for code clarity, so calls like this would look like:

`VariableElementTypeLoader.BuiltInTypes.LOGIC`